scripts/release.sh

```diff
@@ -28,6 +28,14 @@ else
     exit 1
 fi
 
+if [[ $WORKFLOW_NAME == amr ]]; then
```

Comment:
Can we just use regular tags for this?

Reply:
Oops, this file should not be committed, I'll fix that